tests/e2e/vsphere_volume_expansion.go

```diff
@@ -2010,3 +2147,30 @@ func sizeInMb(size resource.Quantity) int64 {
 	actualSize = actualSize / (1024 * 1024)
 	return actualSize
 }
+
+// getFSSizeMbOnSVC returns filesystem size of POD in Mb , which is created on supervisor
+func getFSSizeMbOnSVC(f *framework.Framework, pod *v1.Pod, namespace string) (int64, error) {
```

Comment:
if some change to `getFSSizeMb()` is required, modify it accordingly, instead of writing a new method

Reply:
I have modified the code  getFSSizeMb() , Removed getFSSizeMbOnSV()
Updated all the related tests